Tasks/AzureFileCopyV3/PreJobExecutionAzureFileCopy.ts

```diff
@@ -27,7 +27,7 @@ async function run(): Promise<void> {
         const taskManifestPath = path.join(__dirname, "task.json");
         tl.debug("Setting resource path to " + taskManifestPath);
         tl.setResourcePath(taskManifestPath);
-        let connectionType = tl.getInput('ConnectedServiceNameSelector', true);
+        let connectionType = tl.getInput('ConnectedServiceNameSelector', false);
```

Comment:
Why?

Reply:
Actually, it is an optional parameter and there is an "`if`" control afterwards for checking this variable. But `true` parameter is causing exception. It is probably a copy-paste code from previous Task version.
(It is an existing bug in AzureFileCopy in Prejob stage)